ReClass.NET/Nodes/BaseHexNode.cs

```diff
@@ -101,7 +101,7 @@ public void Update(HotSpot spot, int maxId)
 		{
 			Contract.Requires(spot != null);
 
-			Update(spot);
+			base.Update(spot);
```

Comment:
Strange, why did I remove this a month ago?!

Reply:
Dunno, but it crashes me 🤔